src/utilities/calculator.js

```diff
@@ -1,4 +1,9 @@
-const calculate = (productPrice, data, params, percentage_discount) => {
+const calculate = (productPrice, data, params, percentage_discount, no_of_orders) => {
+  const interest = {
```

Comment:
Can we just use a list? [3.5, 3, 2.75]

Reply:
yes but then it is better to use obj, incase we want to add more interest rate, the keys in that obj represents the length of the existing customer orders

Reply:
We can update then but a list looks like the right answer for this